public/app/features/serviceaccounts/ServiceAccountsListPage.tsx

```diff
@@ -74,8 +74,12 @@ const ServiceAccountsListPage = ({
 
   const onRoleChange = async (role: OrgRole, serviceAccount: ServiceAccountDTO) => {
     const updatedServiceAccount = { ...serviceAccount, role: role };
-    await updateServiceAccount(updatedServiceAccount);
-    await fetchServiceAccounts();
+    updateServiceAccount(updatedServiceAccount);
```

Comment:
Shouldn't it be with `await`? Otherwise there might be a race condition and fetchServiceAccounts may return old result.

Reply:
good catch! will fix that